src/votekit/elections/election_types/ranking/alaska.py

```diff
@@ -0,0 +1,167 @@
+from .abstract_ranking import RankingElection
+from ....pref_profile import PreferenceProfile
+from ...election_state import ElectionState
+from ....utils import first_place_votes, remove_cand
+from ..ranking import Plurality, STV
+from ...transfers import fractional_transfer
+from ....ballot import Ballot
+from typing import Optional, Callable, Union
+from fractions import Fraction
+
+
+class Alaska(RankingElection):
+    """
+    Alaska election. Election method that first runs a Plurality election to choose a user-specified
+    number of final-round candidates, then runs STV to choose :math:`m` winners.
+
+    Args:
+        profile (PreferenceProfile): Profile to conduct election on.
+        m_1 (int, optional): Number of final round candidates, i.e. number of winners of Plurality
+            round. Defaults to 2.
+        m_2 (int, optional): Number of seats to elect in STV round, i.e. number of overall winners.
+            Defaults to 1.
+        transfer (Callable[[str, Union[Fraction, float], Union[tuple[Ballot], list[Ballot]], int], tuple[Ballot,...]], optional):
+            Transfer method. Defaults to fractional transfer.
+            Function signature is elected candidate, their number of first-place votes, the list of
+            ballots with them ranked first, and the threshold value. Returns the list of ballots
+            after transfer.
+        quota (str, optional): Formula to calculate quota. Accepts "droop" or "hare".
+            Defaults to "droop".
+        simultaneous (bool, optional): True if all candidates who cross threshold in a round are
+            elected simultaneously, False if only the candidate with highest first-place votes
+            who crosses the threshold is elected in a round. Defaults to True.
+        tiebreak (str, optional): Tiebreak method to use. Options are None, 'random', and 'borda'.
+            Defaults to None, in which case a tie raises a ValueError.
+
+    """  # noqa
+
+    def __init__(
+        self,
+        profile: PreferenceProfile,
+        m_1: int = 2,
+        m_2: int = 1,
+        transfer: Callable[
+            [str, Union[Fraction, float], Union[tuple[Ballot], list[Ballot]], int],
+            tuple[Ballot, ...],
+        ] = fractional_transfer,
+        quota: str = "droop",
+        simultaneous: bool = True,
+        tiebreak: Optional[str] = None,
+    ):
+        self.m_1 = m_1
+        if m_2 <= 0:
+            raise ValueError("m must be positive.")
```

Comment:
We should do a bunch of validation here. Both `m_1` and `m_2` should be positive and `m_2` has to be <= `m_1` right?